### DISCUSSION TITLE
How to expose the web app8080 port, can not find this configuration where. version 9.7/ no docker

### DISCUSSION THREAD

**Discussion by mumubuku** [apache/skywalking#11736] [Q&A]
2024-01-08 22:24:34,990 - com.linecorp.armeria.server.Server - 832 [armeria-boss-http-*:8080] INFO  [] - Serving HTTP at /[0:0:0:0:0:0:0:0%0]:8080 - http://127.0.0.1:8080/

**Comment by wu-sheng** [comment]
What do you mean leak?

**Reply by mumubuku** [reply]
就是我在服务器上装了个skywalking，不知道怎么外网访问8080端口。2024-01-08 22:24:34,990 - com.linecorp.armeria.server.Server - 832 [armeria-boss-http-*:8080] INFO  [] - Serving HTTP at /[0:0:0:0:0:0:0:0%0]:8080 - http://127.0.0.1:8080/

**Reply by mumubuku** [reply]
I installed skywalking on the server, launched the webapp, and the local request was successful. But can not find the configuration how to access the Internet. A search of some Chinese blogs and videos did not find a solution.

**Reply by wu-sheng** [reply]
That is your basic knowledge about how to use linux or some OS. That is not our responsibility to guide.

**Comment by Superskyyy** [comment]
It listens at 8080 on your machine host network or your container network depending on how you installed SkyWalking. 

Exposing it to the non-host internet is just a matter of configuration of your firewall policies etc. whatever prevents it from being accessed outside of the server, check netstat for more info. It's just like any other web service, nothing is special on SkyWalking side.

**Reply by mumubuku** [reply]
Sorry, I had always thought that SkyWalking requires a configuration to enable external access. After careful review, I realized that it’s because mobile cloud servers require filing for a ICP record before opening port 8080. Changing the port should solve the issue.
